spec/infinispan_local_spec.js

```diff
@@ -81,6 +81,15 @@ describe('Infinispan local client', function() {
     .catch(t.failed(done))
     .finally(done);
   });
+  it('fails when non-configured cache is accessed', function(done) {
+      t.client(t.local, {cacheName: 'unknownCache'})
+          .then(function() {
+              done(new Error('Exception should be thrown while accessing not-configured cache.'));
+         }).catch(function(error) {
+              expect(error).toBe("org.infinispan.server.hotrod.CacheNotFoundException: Cache with name 'unknownCache' not found amongst the configured caches");
+              t.failed(done);
```

Comment:
Should be just `done()`. This is the expected path. As a result, `.finally(done)` is not needed.

Reply:
@galderz Fixed.